content/docs/_index.md

```diff
@@ -10,7 +10,12 @@ toc: false
 
 This section gets you started with Paketo Buildpacks using Paketo [**Builders**][builders], the **pack** CLI, and **Docker**.
 
-Select your favorite language and learn how to get started with Paketo Buildpacks.
+**_Select your favorite language and learn how to get started with Paketo
+Buildpacks._**
+
+Don't see your favorite language below? See the How To guides for even
+more languages, like [Golang][howto/Golang], [Ruby][howto/Ruby], or
+[PHP][howto/PHP].)
```

Comment:
There is a missing `(` and I am not entirely sure where it would go.

Reply:
oop, just a straggler...

Reply:
```suggestion
[PHP][howto/PHP].
```